etna/transforms/outliers/__init__.py

```diff
@@ -0,0 +1,5 @@
+from etna.transforms.outliers.base import OutliersTransform
```

Comment:
If it does not intended for users it should have underscore before its name

Reply:
users can use it with their own outliers detection methods, it's not a private class